mep/books/templates/books/item_list.html

```diff
@@ -16,7 +16,21 @@
 <section class="books search-form">
     {% include 'snippets/breadcrumbs.html' %}
     <form id="books-form">
-        <span class="total-results">{{ items.count|intcomma }} total result{{ items.count|pluralize }}</span>
+        {% if form.errors %}
```

Comment:
Can we pull the form error reporting into a template snippet? It looks the same as the error reporting we have on the member list and presumably we'll need it on the cards as well.

Reply:
Good call!